backend/src/Repository/CaptainFinancialSystemDetailEntityRepository.php

```diff
@@ -93,4 +94,53 @@ public function getCaptainFinancialSystemDetailCurrent(int $captainId): ?array
 
         return $query->getQuery()->getOneOrNullResult();
     }
+
+    public function getLatestFinancialCaptainSystemDetailsForAdmin(int $captainId): ?array
```

Comment:
I guess the function even when doesn't find any records, it will return empty array and not null.